setup.py

```diff
@@ -12,7 +12,7 @@
     'python-dotenv',
     'requests >= 2.20.0, <= 2.25.1',
     'fstrings',
-    'azure-cli-core >= 2.25.0',
+    'azure-cli-core >= 2.34.1, < 2.35.0',
```

Comment:
>'azure-cli-core >= 2.34.1, < 2.35.0', [](http://example.com/codeflow?start=4&length=37)

I copied this from the PR description, it would be great to add this info for future context for this line.
"Please note >=2.35.0
cannot be used as this version is not compatible with the docker
dependency, since docker requires websocket-client==0.56.0 and
azure-cli-core>=2.35.0 requires websocket-client==1.31.1."